### ISSUE TITLE
Opus transcoding - bitrates other that 48khz and Forward Error Correction

### ISSUE THREAD

**Issue by marcinkowalczyk** [sipwise/rtpengine#528]
Hi,

 Is there a plan to support bandwidth other than 48khz and Forward Error Correction?

Regards
Marcin

**Comment by rfuchs** [comment]
Haven't looked into it yet, but I assume these can be enabled by passing the right options to ffmpeg, e.g. `packet_loss` to enable FEC and `cutoff` for various bandwidth constraints. See https://ffmpeg.org/ffmpeg-codecs.html#libopus-1

If these are what is required, then it would be very simple to enable them. `opus_set_options()` would be the right spot.

**Comment by marcinkowalczyk** [comment]
Yes, general having all Opus/ffmpeg options would be very handy.

**Comment by firestation** [comment]
Hi, i have checked "packet_loss" and rtpengine cases. 
i am used 11.1.0.5 version. What i have found: 
1. libopus with ffmpeg 4.2.7 does not have fec option, need to apply path to add it. 
2. rtpengine does not send codec private options like packet_loss to libopus, yu can send any of global ffmpeg options with addional slashes like "codec-transcode-opus//////compression_level--1", but if you try send "codec-transcode-opus//////packet_loss--10" you will get "Option not found" message from avlog. 
I have to set: 
                codeclib_set_av_opt_int(enc, "packet_loss", 30);
                codeclib_set_av_opt_int(enc, "fec", 1);
into lib/codeclib.c to send this options to ffmpeg. 
I am not sure i am did it in right way. But if yes, then some issue is present when rtpengine parsing options for codecs. 

For me, there is one question is still misunderstanding, if i set packet_loss option, then i can not feel any additional profit in case of packets loss on the endpoint side.
